### ISSUE TITLE
`npm install -g firedoc` cannot install `scripts` folder

### ISSUE THREAD

**Issue by nantas** [fireball-x/firedoc#23]
I tried 

``` bash
npm install -g firedoc
# and
npm install -g fireball-x/firedoc
```

In both case, there's no `scripts` folder installed in `/Users/username/.nvm/versions/node/v0.12.0/lib/node_modules/firedoc`

Thus theme installing command line will fail.

**Comment by yorkie** [comment]
This seems to be fixed at `firedoc@1.5.5` :)

**Comment by yorkie** [comment]
Verified at my PC, it works and closing now :)
